packages/esm-patient-vitals-app/src/vitals-and-biometrics-header/vitals-header.scss

```diff
@@ -11,11 +11,17 @@
   justify-content: space-between;
   align-items: center;
   min-height: layout.$spacing-09;
-  margin: layout.$spacing-03 0 0 layout.$spacing-05;
+  padding-left: layout.$spacing-05;
   outline: none;
   pointer-events: visibleFill;
 }
 
+.emptyStateVitalsHeader {
+  @extend .vitalsHeader;
+  background-color: $openmrs-background-grey;
+  border-bottom: 1px solid $ui-03;
+}
```

Comment:
I've taken the liberty of pushing a commit that fixes the background color of the empty state:

### Before (in the UAT distro)

![CleanShot 2024-09-24 at 9  47 24@2x](https://github.com/user-attachments/assets/729d1f51-1cd5-40bf-81ac-8d9b2f8ab9f6)

### After

![CleanShot 2024-09-24 at 9  36 40@2x](https://github.com/user-attachments/assets/ad9bd2fc-0bfb-4c93-b58c-aa30c230b184)

Reply:
Do the changes look reasonable to you?

Reply:
This look great. Thanks!